addon/components/o-s-s/checkbox.ts

```diff
@@ -0,0 +1,54 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { guidFor } from '@ember/object/internals';
+
+const AVAILABLE_SIZES = ['sm'];
+
+interface OSSCheckboxArgs {
+  checked: boolean;
+  disabled: boolean;
+  size: string;
+  onChange(value: boolean): void;
+}
+
+export default class OSSCheckbox extends Component<OSSCheckboxArgs> {
+  constructor(owner: unknown, args: OSSCheckboxArgs) {
+    super(owner, args);
+
+    if (typeof args.checked !== 'boolean') {
+      throw new Error(`[component][OSS::Checkbox] Boolean @checked argument is mandatory.`);
+    }
+
+    if (!args.onChange) {
+      throw new Error(`[component][OSS::Checkbox] @onChange argument is mandatory.`);
```

Comment:
Is it really though ? :trollface: 
Does the 2way binding work with this component ? 😇

Reply:
The `checked` arg is never actually updated inside the component, that's why